setup.py

```diff
@@ -129,40 +133,81 @@ def run(self):
         SDist.run(self)
 
 
-with open('requirements.txt') as requirements_file:
-    install_requirements = requirements_file.read().splitlines()
-    if not install_requirements:
-        print("Unable to read requirements from the requirements.txt file"
-              "That indicates this copy of the source code is incomplete.")
+def read_file(file_name):
+    """Read file and return its contents."""
+    with open(file_name, 'r') as f:
+        return f.read()
```

Comment:
I usually feel that something of this level of granularity is overly subdividing the code but this is setup.py, so I'm not as worried about it here as in the main code.

Reply:
Normally I feel similar way, but after seeing lots of `setup.py`s, which are turned into understandable magic sets I just can't ignore this.